policyengine_us/parameters/gov/states/nh/tax/income/main.yaml

```diff
@@ -0,0 +1,14 @@
+description: New Hapshire provides this tax rate on bonds, notes, and money at interest.
+values:
+  2022-01-01: 0.05
+  2023-01-01: 0.04
+  2024-01-01: 0.03
+  2025-01-01: 0.02
+  2026-01-01: 0.01
+metadata:
+  unit: /1
+  period: year
+  label: New Hampshire tax rate
+  reference:
+    - title: New Hampshire Statutes - TITLE V - CHAPTER 77 - Section 77:1
```

Comment:
Could you add a reference that specifies the tax base (interest and dividends)? eg tax form